geoutils/georaster.py

```diff
@@ -1167,20 +1170,44 @@ def coords(self, offset='corner', grid=True):
         else:
             return xx[:-1], yy[:-1]
 
-    def xy2ij(self,x,y):
+    def xy2ij(self,x,y, op=math.floor,shift_area_or_point=False):
         """
         Return row, column indices for a given x,y coordinate pair.
 
         :param x: x coordinates
         :type x: array-like
         :param y: y coordinates
         :type y: array-like
+        :param op: operator to calculate index
+        :type op: operator
+        :param shift_area_or_point: shift according to AREA_OR_POINT GDAL attribute:
+         interpretation to what the raster value corresponds to (AREA = lower left or POINT = center)
```

Comment:
area is actually upper left

Reply:
You mean upper left in y/x dimension? Because y is inverted? So in indexes: Area = (0,0) and Point = (0.5,0.5), right? The new tests work that way.

Reply:
Yes, y axis is downward, so (0,0) is the upper left corner.

Reply:
fixed in 37da8c2